tests/ff-server-sdk-test/api/analytics/AnalyticsManagerTest.cs

```diff
@@ -23,145 +24,239 @@ public void Should_add_single_evaluation_for_single_feature_to_analytics_cache()
             var analyticsCacheMock = new AnalyticsCache();
             var connectorMock = new Mock<IConnector>();
             var analyticsPublisherServiceMock = new AnalyticsPublisherService(connectorMock.Object, analyticsCacheMock, new NullLoggerFactory());
-
+        
```

Comment:
Not too sure wha's going on with these extra lines

Reply:
Rider will insert an indent everytime you press enter. You can strip them out with

<img width="458" alt="image" src="https://github.com/harness/ff-dotnet-server-sdk/assets/115623869/2bca49ce-3a20-413a-83ce-9156d7ec6e47">